client/reader/components/quick-post/style.scss

```diff
@@ -16,20 +16,25 @@
 		display: flex;
 		justify-content: space-between;
 		align-items: end;
+		gap: 10px;
 
-		.sites-dropdown__wrapper {
-			border-color: var( --color-border-subtle );
-			border-radius: 6px;
+		.sites-dropdown {
+			flex: 0 1 300px;
+
+			&__wrapper {
+				border-color: var( --color-border-subtle );
+				border-radius: 6px;
+			}
+
+			&.is-open .sites-dropdown__wrapper {
+				border-radius: 6px 6px 0 0;
+			}
```

Comment:
Non blocking minor nitpick. While we are reorganizing these we could combine these 2.  I think we could probably move the latter rule nested into the former with just `&.is-open`

Reply:
Good call! I simplified the definition in https://github.com/Automattic/wp-calypso/pull/102238/commits/fccfbb3694d7960a05a12e1881bcd79ec4fa4fd8, thanks!